velox/exec/tests/UnnestTest.cpp

```diff
@@ -39,6 +39,19 @@ TEST_F(UnnestTest, basic) {
   assertQuery(op, "SELECT c0, UNNEST(c1) FROM tmp WHERE c0 % 7 > 0");
 }
 
+TEST_F(UnnestTest, basicMap) {
```

Comment:
Consider creating DuckDB table with 3 columns, c0, c1.keys and c1.values, then unnesting 2 array columns in the query.

Reply:
Yeah, this approach worked. Thanks for the suggestion.

Have also added another test for empty and null maps.